src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -255,3 +276,55 @@ def load_arguments(self, _):
+                           help='The minimal workernode count for Load-based atuoscale.')
+                c.argument('max_count', arg_group='Load-based Autoscale',
+                           help='The max workernode count for Load-based atuoscale.')
+                c.argument('timezone', arg_group='Schedule-based Autoscale', validator=validate_timezone_name,
+                           completer=get_generic_completion_list(AUTOSCALE_TIMEZONES),
+                           help='The timezone for schedule autoscale type. '
+                                'Values from `az hdinsight autoscale list-timezones`')
+
+        with self.argument_context('hdinsight autoscale create') as c:
+            c.argument('type', arg_type=get_enum_type(['Load', 'Schedule']), help='The autoscale type.')
+            c.argument('days', nargs='+', arg_group='Schedule-based Autoscale',
+                       completer=get_generic_completion_list(week_days),
+                       help='A space-delimited list of schedule day. Valid days are {}.'.format(', '.join(week_days)))
+            c.argument('time', arg_group='Schedule-based Autoscale', validator=validate_time,
+                       help='The 24-hour time in the form xx:xx in days.')
+            c.argument('count', arg_group='Schedule-based Autoscale', help='The schedule workernode count.')
+
+        # autoscale condition
+        autoscale_condition_commands = ['create', 'update', 'delete', 'list']
+        for command in autoscale_condition_commands:
+            with self.argument_context('hdinsight autoscale condition ' + command) as c:
+                c.argument('cluster_name', options_list=['--cluster-name'],
+                           completer=get_resource_name_completion_list('Microsoft.HDInsight/clusters'),
+                           help='The name of the cluster.')
+
+        for command in ['create', 'update']:
+            with self.argument_context('hdinsight autoscale condition ' + command) as c:
+                c.argument('days', nargs='+', arg_group=None, completer=get_generic_completion_list(week_days),
+                           help='A space-delimited list of schedule day. '
+                                'Valid days are {}.'.format(', '.join(week_days)))
+                c.argument('time', arg_group=None, validator=validate_time,
+                           help='The 24-hour time in the form xx:xx in days.')
+                c.argument('count', arg_group=None, help='The schedule workernode count.')
+        for command in ['create', 'update']:
+            with self.argument_context('hdinsight autoscale condition ' + command) as c:
+                c.argument('index', help='The schedule condition index which starts with 0.')
+
+        with self.argument_context('hdinsight autoscale condition delete') as c:
+            c.argument('index', nargs='+',
+                       help='The Space-separated list of condition indices which start with 0 to delete.')
```

Comment:
starts with

Reply:
I am not sure whether it should be "starts with", because it is condition indices.

---
In reply to: [466368703](https://github.com/Azure/azure-cli/pull/14692#discussion_r466368703) [](ancestors = 466368703)

Reply:
list is singular and so it should be "starts with" (if there were multiple lists then it should be "start with")

---
In reply to: [466800733](https://github.com/Azure/azure-cli/pull/14692#discussion_r466800733) [](ancestors = 466800733,466368703)

Reply:
fixed.

---
In reply to: [467171759](https://github.com/Azure/azure-cli/pull/14692#discussion_r467171759) [](ancestors = 467171759,466800733,466368703)